### PR TITLE
Added app controls section

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-2397](https://team-1624093970686.atlassian.net/browse/AR-2397).

## Changes
- Added Delete button
- Added Confirm delete and progress timer for delete popups
- Added logic for timer. Circular Progress timer triggers every 10ms and actual display timer triggers every 1000ms
- Added Deletion logic
- Added Full screen loader while consuming delete API

## Screenshots
Screenshot from 2022-05-05 13-02-35
Screenshot from 2022-05-05 13-02-45
Screenshot from 2022-05-05 13-02-51
Screenshot from 2022-05-05 13-03-51

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: src/components/app-configure/general/DeleteTimerPopup.vue [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      :percent="timerProgressPercentage"
+      :border-width="5"
+      :border-bg-width="5"
+      fill-color="#13A3FD"
+      empty-color="transparent"
+      :size="200"
+    />
+    <div class="flex inner-clock">
+      <span class="sub-heading-2">
+        00:{{ timer.toString().padStart(2, '0') }}
```
Comment (ajithranka) [inline src/components/app-configure/general/DeleteTimerPopup.vue:55]: Move expression to a computed property.

File: src/components/app-configure/general/DeleteTimerPopup.vue [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+function handleCancelDelete() {
+  clearTimer()
+  emit('cancel')
+}
+
+function proceedDelete() {
+  clearTimer()
+  emit('delete')
+}
```
Comment (ajithranka) [inline src/components/app-configure/general/DeleteTimerPopup.vue:39]: @shrinathprabhu @karthik-durai 

Let's follow a consistent naming convention for event handlers. Few possible options:

For `event`, name handler as `handle[Event]`
```
@click="handleCancellation"
@click="handleProceed"
```

For `event`, name handler as `[event]EventHandler`:
```
@click="cancelEventHandler"
@click="proceedEventHandler"
```

Which one (or any other) should we adopt?

Comment (shrinathprabhu) [reply src/components/app-configure/general/DeleteTimerPopup.vue:39]: I would prefer handle[Event]

File: src/components/app-configure/general/AppControls.vue [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        />
+        <DeleteTimerPopup
+          v-if="deleteTimerPopup"
+          @cancel="deleteTimerPopup = false"
+          @delete="deleteApp"
+        />
+      </VCard>
+    </VOverlay>
+  </section>
+  <FullScreenLoader v-if="isLoading" :message="loadingMessage" />
```
[the remark this answers is not in the document]
Comment (shrinathprabhu) [reply src/components/app-configure/general/AppControls.vue:66]: Yeah we can do that

**Review by ajithranka — APPROVED** [review]

**Pull request merged**